examples/StandardFirmataWiFi/wifiConfig.h

```diff
@@ -140,11 +153,11 @@ char wep_key[] = "your_wep_key";
  * CONFIGURATION ERROR CHECK (don't change anything here)
  *============================================================================*/
 
-#if ((defined(ARDUINO_WIFI_SHIELD) && (defined(WIFI_101) || defined(HUZZAH_WIFI))) || (defined(WIFI_101) && defined(HUZZAH_WIFI))  || (defined(WIFI_101) && defined(ESP_WIFI))  || (defined(ESP_WIFI) && defined(HUZZAH_WIFI)) || (defined(ESP_WIFI) && defined(ARDUINO_WIFI_SHIELD)))
+#if ((defined(ARDUINO_WIFI_SHIELD) && (defined(WIFI_101) || defined(HUZZAH_WIFI))) || (defined(WIFI_101) && defined(HUZZAH_WIFI))  || (defined(WIFI_101) && defined(ESP8266_WIFI))  || (defined(ESP8266_WIFI) && defined(HUZZAH_WIFI)) || (defined(ESP8266_WIFI) && defined(ARDUINO_WIFI_SHIELD)))
```

Comment:
For ARDUINO_WIFI_SHIELD and WIFI_101 one could provide the two include alternatives and use only one define.

I assume the HUZZAH_WIFI define is reserved for the CC3000 WiFi shield. I assume it comes with its own library, but I don't know if it is identical with WiFi/WiFi101. The HUZZAH ESP8266 is covered by the ESP8266.

With the ESP8266 one could use the architecture define already in use in Boards.h. This could replace the ESP_WIFI define and could automatically exclude all shields.